integration_tests/tests/s3-invoice-raw-store-tests.ts

```diff
@@ -23,75 +25,60 @@ describe("\n Happy path - Upload valid mock invoice pdf to the raw invoice pdf b
   const standardisedFolderPrefix = "btm_billing_standardised";
   const invoice = randomInvoice();
   test("raw-invoice-textract-data and storage buckets should contain textracted and standardised data file for uploaded valid pdf file in raw-invoice-pdf bucket and should move the original raw invoice to successful folder in s3 raw-invoice-pdf bucket", async () => {
-    const s3Object = await createInvoiceInS3(invoice);
-    const checkRawPdfFileExists = await checkIfS3ObjectExists(s3Object);
-    expect(checkRawPdfFileExists).toBeTruthy();
-
-    const checkTextractDataFileContainsStringFromOriginalPdf =
-      async (): Promise<boolean> => {
-        const textractedFileContents =
-          await getS3FileContentsBasedOnLastModified(
-            testStartTime,
-            textractBucket
-          );
-        return textractedFileContents.some((file) =>
-          file?.includes(invoice.invoiceNumber)
-        );
-      };
+    const filename = `raw-Invoice-${Math.random()
+      .toString(36)
+      .substring(2, 7)}-validFile`;
+    const s3Object = await createInvoiceInS3(invoice, `${filename}.pdf`);
+    const isInvoiceUploadSuccessful = await checkIfS3ObjectExists(s3Object);
+    if (!isInvoiceUploadSuccessful) {
```

Comment:
Feels like this condition should be after the poll step below which waits for invoice to be uploaded to raw bucket